docs/source/whats-new.md

```diff
@@ -4,6 +4,47 @@ What's new
 See [GitHub releases page](https://github.com/OSOceanAcoustics/echopype/releases) for the complete history.
 
 
+# v0.8.2 (2023 November 20)
+
+## Overview
+
+This release includes a number of important performance enhancements, bug fixes, and under-the-hood refactoring to improve code readability.
+
+## Enhancements
+- Full refactor of the functionality to directly storing parsed data to zarr before set_groups_* (#1185, #1218 [NOT MERGED], #1070)
+  - This avoids memory expansion during NaN padding across channel
+- Improve `compute_MVBS` using flox (#1124)
+- Revive and improve `compute_NASC` using flox (#1167)
+- Refactor AZFP XML parser
+  - Parse more parameters and store in the resulting `EchoData` objects (#1135)
+  - Add support for AZFP multiple phase settings (#1182)
+- AZFP pressure data handling
+  - Parse and store AZFP pressure data (#1189)
+  - Add `depth_from_pressure` method and store calculated depth data in Platform group (#1207, #1202 [NOT MERGED])
```

Comment:
The `depth_from_pressure` function is a generic utility function and is not tied to AZFP. So, I think it should be its own item:

```suggestion
  - Add `depth_from_pressure` utility function to calculate depth from pressure and latitude (#1207)
- AZFP pressure data handling
  - Parse and store AZFP pressure data in the Environment group (#1189)
  - Use pressure to calculate depth, stored in `vertical_offset` in the Platform group (#1202) [NOT MERGED]
```

Reply:
I am a bit hesitate regarding converting pressure to depth directly, because of the uncertainty in latitude. I'll comment on it in #1202.